src/HtmlUtils.tsx

```diff
@@ -553,11 +553,11 @@ export function bodyToHtml(content: IContent, highlights: Optional<string[]>, op
                 sanitizeParams.textFilter = function (safeText) {
                     if (opts.emotes) {
                         return highlighter
-                            .applyHighlights(safeText, safeHighlights)
+                            .applyHighlights(safeText, safeHighlights!)
                             .join("")
-                            .replace(CUSTOM_EMOTES_REGEX, (m) => (opts.emotes.get(m) ? opts.emotes.get(m) : m));
+                            .replace(CUSTOM_EMOTES_REGEX, (m) => (opts?.emotes?.get(m) ? opts?.emotes?.get(m)! : m));
```

Comment:
the condition side could/should use `has`

Reply:
done